pages/common/openttd.md

```diff
@@ -0,0 +1,28 @@
+# openttd
+
+> Open source clone of the Microprose game "Transport Tycoon Deluxe".
+> More information: <https://www.openttd.org>.
+
+- Load new/save game at start:
```

Comment:
I'd split that in two examples:
* Load game → `openttd -g {{path/to/file}}`
* Start new game → `openttd -g`

Reply:
To my knowledge, it also creates that savefile, so why split?

Reply:
Oh, then I'd update the description, because it's really confusing now. How about:

> Load a save file or start a new game if the file doesn't exist

Reply:
What's confusing about `{{new|existing}}` ie; `new/save` ?

Reply:
It doesn't really sound right. @bl-ue I need an expert here.

Reply:
I'd say something like:
```suggestion
- Start a game from a file, creating the file if it doesn't exist:
```
That's more in tune with some of our other pages.

Reply:
I just tested it and if you use a file that doesn't exist, you get an error.

> I'd split that in two examples:
> 
>     * Load game → `openttd -g {{path/to/file}}`
> 
>     * Start new game → `openttd -g`

Reply:
Ah, good to know.  Then @Waples I'd split it into two examples as @marchersimon suggests.